packages/plugins/plugin-debug/src/translations.ts

```diff
@@ -50,7 +50,7 @@ export const translations = [
         'space label': 'Space',
         'feeds label': 'Feeds',
         'objects label': 'Objects',
-        'schema label': 'Schema',
+        'schema label': 'Record',
```

Comment:
imo debug plugin should stay saying schema for now, just as it says "objects"

Reply:
I see, fair enough